cmake/doxygen.cmake

```diff
@@ -31,27 +31,27 @@ macro(build_doxygen_documentation)
     # set the destination folder to be devel/share/[project_name]/doc/
     set(doc_build_folder ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION})
     set(doc_install_folder ${CMAKE_INSTALL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION})
-    
+
     # Create the doxyfile in function of the current project.
     # If the Doxyfile.in does not exists, the cmake step stops.
     if(NOT ${mpi_cmake_modules_SOURCE_PREFIX} STREQUAL "")
         set(RESOURCE_FOLDER ${mpi_cmake_modules_SOURCE_PREFIX}/resources)
-    elseif(NOT ${mpi_cmake_modules_SOURCE_DIR} STREQUAL "")
-        set(RESOURCE_FOLDER ${mpi_cmake_modules_SOURCE_DIR}/resources)
+    elseif(NOT ${MPI_CMAKE_MODULES_RESOURCES_DIR} STREQUAL "")
```

Comment:
With your changes we can directly use `MPI_CMAKE_MODULES_RESOURCES_DIR` in the config below no? This if was in order to check if the SOURCE_DIR or the SOURCE_PREFIX variable was set or not...

Reply:
:+1: